pkg/slayers/path/scion/base_spec.gobra

```diff
@@ -44,6 +45,13 @@ pred (b *Base) Mem() {
 	(0 < b.NumINF ==> 0 < b.NumHops)
 }
 
+ghost 
+ensures res
+decreases
+pure func validMetaLenInPath() (res bool) {
+	return MetaLen == path.MetaLen
+}
```

Comment:
where is this used?

Reply:
I created this function to make sure that the new ghost variable `path.MetaLen` is equal to the original variable but I am not really sure how to use this function. I though if I ensure the result, it implies that the variables are equal?

Reply:
ah I see. I would put this in file base_spec_test.gobra to make it clear that it is a test (and not a member that we use somewhere)

Reply:
Alright, function is moved. It is ready to merge.